src/proto/thread_telemetry.proto

```diff
@@ -513,6 +513,69 @@ message TelemetryData {
     optional bool has_others_route_added = 3;
   }
 
+  enum EpskcState {
+    EPSKC_STATE_READY_FOR_CONNECTION = 0;
+    EPSKC_STATE_CONNECTED = 1;
+    EPSKC_STATE_INACTIVE = 2;
```

Comment:
numbering inactive state firstly as it's generally the default state

Reply:
Updated (BorderAgentState)